review.md

```diff
@@ -2,48 +2,73 @@
-1. If the listing of an existing package is modified, then the moderator must verify that the new information is complete and correct. In many cases, it may be necessary to obtain permission from the package maintainer.
-1. The reasons for deleting a package listing may vary on a case-by-case basis. The moderator must carefully consider the impact that deletion would have on the community and on reverse dependencies. In many cases, it may be necessary to obtain permission from the package maintainer.
+This policy dictates the process by which R-multiverse reviews and accepts contributed R packages.
+All contributions must comply with R-multiverse policies, including but not limited to [Acceptable Use](https://r-multiverse.org/aup.html), [Terms of Use](https://r-multiverse.org/terms.html), and [Code of Conduct](https://r-multiverse.org/conduct.html).
+
+## How review works
+
+When it reviews a [new pull request](https://github.com/r-multiverse/contributions/pulls), the bot makes one of three choices:
+
+1. Merge the pull request to accept the contribution.
+2. Close the pull request to reject the contribution.
+3. Flag the pull request for manual review by a [moderator](https://r-multiverse.org/governance.html#moderator).
+
+## Automatic acceptance
+
+The bot automatically accepts the contribution if the [pull request](https://github.com/r-multiverse/contributions/pulls):
+
+1. Was [created by the GitHub web interface](https://r-multiverse.org/contributors.html).
+1. Adds new contributed listings to the [`packages` folder](https://github.com/r-multiverse/contributions/tree/main/packages).
+1. Does not add, modify, or delete any other files in <https://github.com/r-multiverse/contributions>.
+
+and if each new [contributed listing](https://github.com/r-multiverse/contributions/tree/main/packages):
+
+1. Is a single line of text with a valid HTTPS URL.
+1. Points to an existing public GitHub/GitLab repository.
+
+and if the contributed GitHub/GitLab repository:
+
+1. Includes an R package at the top level whose package name is the same as the repository name.
+1. Includes a license from the "Recommended licenses" section at the end of this policy.
+1. Includes a GitHub/GitLab [release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases).
+1. Is not part of the CRAN mirror at <https://github.com/cran>.
+1. Is mentioned in the `URL` field its CRAN page (if on CRAN).
+
+and if the author of the [pull request](https://github.com/r-multiverse/contributions/pulls):
+
+1. Is a [public member](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership) of one of the trusted [GitHub organizations](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations) listed at <https://github.com/r-multiverse/contributions/blob/main/organizations>.
```

Comment:
the last link is broken

Reply:
Just added an new org list at https://github.com/r-multiverse/contributions/blob/main/organizations. Many more to come, but it's a start.